_includes/segment/navbar.html

```diff
@@ -0,0 +1,10 @@
+<div class="ui container nav">
+	<div class="{% if page.page_name == "adoption" %}four{% else %}three{% endif %} ui buttons">
+		<a href="/?skipToListings" class="ui blue button{% if page.page_name == "home" %} active-nav {% endif %}" data-tooltip="Merchants accepting Bitcoin Cash" data-position="top center" data-inverted="">Main</a>
+		<a href="/donations/?skipToListings" class="ui violet button{% if page.page_name == "donations" %} active-nav {% endif %}" data-tooltip="Organizations accepting Bitcoin Cash donations" data-position="top center" data-inverted="">Donations</a>
+		<a href="/adult/?skipToListings" class="ui pink button{% if page.page_name == "adult" %} active-nav {% endif %}" data-tooltip="Warning! Links to sexually oriented material ahead!" data-position="top center" data-inverted="">Adult</a>
+		{%- if page.page_name == "adoption" -%}
```

Comment:
Should we continue to hide the **Adoption** link on other pages permanently?

Reply:
I put that in as a temporary provision to make sure if we merged this without finishing that page we didnt need to show it until we could circle back to it. if we have everything, we can remove that component